src/ValueHandler/ProductOptionValueHandler.php

```diff
@@ -16,20 +18,24 @@
 use Sylius\Component\Resource\Factory\FactoryInterface;
 use Sylius\Component\Resource\Repository\RepositoryInterface;
 use Sylius\Component\Resource\Translation\Provider\TranslationLocaleProviderInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
 use Webgriffe\SyliusAkeneoPlugin\ValueHandlerInterface;
 use Webmozart\Assert\Assert;
 
 final class ProductOptionValueHandler implements ValueHandlerInterface
 {
     private ?TranslationLocaleProviderInterface $translationLocaleProvider;
 
+    private ?TranslatorInterface $translator;
+
     public function __construct(
         private AkeneoPimClientInterface $apiClient,
         private ProductOptionRepositoryInterface $productOptionRepository,
         private FactoryInterface $productOptionValueFactory,
         private FactoryInterface $productOptionValueTranslationFactory,
         private RepositoryInterface $productOptionValueRepository,
-        TranslationLocaleProviderInterface $translationLocaleProvider = null
+        TranslationLocaleProviderInterface $translationLocaleProvider = null,
+        TranslatorInterface $translator = null
```

Comment:
```suggestion
        TranslatorInterface $translator,
        TranslationLocaleProviderInterface $translationLocaleProvider = null
```

BC break is allowed on master.

Reply:
Yes, but I will keep this type of changes on the #130.

Reply:
Ok!